test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/JwtTokenAuthTests.cs

```diff
@@ -72,6 +73,18 @@ public async Task InvokeAdminApi_InvalidToken_Fails(string headerName)
             Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
         }
 
+        [Fact]
+        public async Task InvokeAdminApi_ValidToken_UTF8Encoding_Succeeds()
```

Comment:
This is just a sanity test since I removed a Convert.FromBase64String conversion above on line 31 and we now use ToKeyBytes in all cases which handles both hex and base64.